pages/docs/running/running-distributed.md

```diff
@@ -44,3 +44,30 @@ If you have logged into remote machine via SSH, then be careful to use a tool su
 This allows to safely detach from a session without the running process getting killed.
 
 If your cluster is managed using SLURM, then [further steps may be necessary](running-slurm).
+
+## Running participants in different systems
```

Comment:
They don't need to be different systems though.  docker containers or VMs are also possible.

https://precice.discourse.group/t/how-to-configure-precice-communication-on-kubernetes-with-tcp-port-access-control/1451

Reply:
In my view, a Docker container is a different system than its host.
Any other name I can think of is unnecessarily technical. Any suggestions?

Reply:
Instead of changing the title, I added coupling to containers and VMs explicitly as a use case, linking to Olivia's post.